index.js

```diff
@@ -24,16 +27,26 @@ var Dropzone = React.createClass({
     activeStyle: React.PropTypes.object,
     className: React.PropTypes.string,
     activeClassName: React.PropTypes.string,
+    rejectClassName: React.PropTypes.string,
 
     disableClick: React.PropTypes.bool,
     multiple: React.PropTypes.bool,
+    accept: React.PropTypes.string,
+  },
+
+  allFilesAccepted: function(files) {
+    return files.every(file => accept(file, this.props.accept))
```

Comment:
Don't you use babel to compile the source? https://github.com/paramaggarwal/react-dropzone/blob/master/package.json#L7

The only thing is you'll probably need to add .babelrc with `"optional": ["runtime"],`